build.js

```diff
@@ -20,23 +20,28 @@ async function build(packageDir) {
     process.exit(1);
   }
 
+  // prettier-ignore
+  const banner =
+`/* * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ *                    Paper Shaders                    *
+ *       https://github.com/paper-design/shaders       *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * */
```

Comment:
Couldn't resist a cooler looking banner when I saw we add one

Reply:
sick I love it